src/main/java/com/dut/pbl6_server/service/impl/ThreadServiceImpl.java

```diff
@@ -176,4 +179,23 @@ public DataWithPage<ThreadResponse> getFollowingThreads(Long userId, Pageable pa
             PageUtils.makePageInfo(page)
         );
     }
+
+    @Override
+    public DataWithPage<ThreadResponse> getThreadSharesByAccount(Account currentUser, Pageable pageable) {
+        Page<Thread> page = null;
+
+        List<Long> threadIds = threadSharersRepository.getListThreadIdByUser(currentUser);
+
+        if (threadIds == null || threadIds.isEmpty()) {
```

Comment:
Cái kiểm tra list rỗng hay null á, Vinh có viết ở CommonUtils rồi á. Dùng cái đó cho tiện
```suggestion
        if (CommonUtils.List.isEmptyOrNull(threadIds)) {
```